frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -266,7 +150,7 @@ export const SidePanelSupport = (): JSX.Element => {
                                     </LemonBanner>
                                 </Section>
                             ) : null}
-
+                            {/* Anthropic Max below, sidelined due to restrictive rate-limits. Commented out for now, but keeping here in case InKeep Max doesn't work out. 
```

Comment:
Just noticed ALL of this code is commented out, I thought it's only the comment.
Let's not commit in commented-out blocks of code, that's a recipe for abandonment and confusion down the line. Removing in a commit